### PR TITLE
Design/Styling fixes

### PR DESCRIPTION
**Pull request by boidushya**
## Title

Yet another PR with styling fixes as per #59 ✨

### Divider color in Account/Avatar dropdown

| Before | After |
| ------------- | ------------- |
| image  | image  |

### Send invite button disabled state

| Mode | Before | After |
| ------------- | ------------- | ------------- |
| Dark | image | image |
| Light | image | image |

### New App plus icon size

| Before | After |
| ------------- | ------------- |
| image | image |

###  Auto navigate to chat on invite

https://user-images.githubusercontent.com/26746725/227567061-57677259-9f72-4c83-8803-8bc3da9be3b3.mov

### Status indicator in hover area in notifications

| Before | After |
| ------------- | ------------- |
| image | image |

Not entirely satisfied with the way this turned out. I'll take a closer look at this later.

###  Checkbox Styling

| State/Mode | What it looks like |
| ------------- | ------------- |
| Unchecked/Dark | image |
| Unchecked/Light | image |
| Checked/Dark | image |
| Checked/Light | image |

### Content Jumping in SVG sidebar items 
Probably like a 1-2px jump (Focus on the Cog icon in before if you don't see it immediately)

#### Before
https://user-images.githubusercontent.com/26746725/227550820-2d93d879-84b6-4432-b87d-b98e034531af.mov 

#### After
https://user-images.githubusercontent.com/26746725/227550932-2d4d0068-fc2d-44d7-a0e7-94cf9e298112.mov

### CODE REVIEW

File: src/components/general/Divider/Divider.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .Divider {
   height: 1px;
   width: 100%;
-  background-color: #ddd;
-  margin: 0.5em 0;
+  background-color: var(--border-color-2);
+  //   margin: 0.5em 0;
```
Comment (devceline) [inline src/components/general/Divider/Divider.scss:5]: We can just remove unused/commented code

File: src/components/general/Dropdown/Dropdown.tsx [hunk 2]
```diff
@@ -1,4 +1,6 @@
 import type { ReactNode } from 'react'
+
+// eslint-disable-next-line no-duplicate-imports
```
Comment (devceline) [inline src/components/general/Dropdown/Dropdown.tsx:3]: Going to make a pr that removes this rule

File: src/components/notifications/AppNotifications/AppNotifications.scss [hunk 3]
```diff
@@ -14,7 +14,7 @@
     gap: 12px;
     width: 100%;
     align-items: center;
-    padding: 8px;
+    padding: 10px 4px;
```
Comment (devceline) [inline src/components/notifications/AppNotifications/AppNotifications.scss:17]: Can we use relative values like `em` here?

**Review by devceline — APPROVED** [review]
LGTM, minor comments. Excellent work

File: src/components/general/Divider/Divider.scss (hunk 1 above)
Comment (boidushya) [reply src/components/general/Divider/Divider.scss:5]: Sure thing!

File: src/components/notifications/AppNotifications/AppNotifications.scss (hunk 3 above)
Comment (boidushya) [reply src/components/notifications/AppNotifications/AppNotifications.scss]: For sure 🫡

File: src/components/general/Checkbox/Checkbox.scss [hunk 4]
```diff
@@ -1,35 +1,45 @@
...
-  width: 1.75em;
-  height: 1.75em;
+  display: grid;
+  place-items: center;
+  box-sizing: border-box;
 
   input[type='checkbox'] {
     appearance: none;
-    background-color: var(--accent-color-1);
+    // background-color: var(--accent-color-1);
```
Comment (Cali93) [inline src/components/general/Checkbox/Checkbox.scss:8]: Can you remove the commented CSS please ?

**Review by Cali93 — APPROVED** [review]
LGTM! 🚀

**Pull request merged**